Investigations/Office Tools.md

```diff
@@ -0,0 +1,204 @@
+
+- [Graph DB](https://blog.capterra.com/free-database-software/#2)
+
+- [MariaDB (Opensource)](https://blog.capterra.com/free-database-software/#3)
+
+- [PostgreSQL (Opensource)](https://blog.capterra.com/free-database-software/#4)
+
+- [QuintaDB](https://blog.capterra.com/free-database-software/#5)
+
+- [Sonadier](https://blog.capterra.com/free-database-software/#6)
+
+- MariaDB is an open source relational database for data storage, data insertion into tables, data modifications, and data retrieval. As an open source solution, the tool is free to use and you can get started by downloading the software on your desktop or laptop.
+
+Pros: Users mention that MariaDB&#39;s open source community consist of active members who roll out patches and updates regularly.
+
+Cons: Users mention that the tool lacks a graphical user interface and users can operate the tool only by using the command line.
+
+- PostgreSQL is an open-source database that comes with features such as data indexing, user configuration settings, data import/export, and version control.
+
+While PostgreSQL is a relational database, it also allows creating NoSQL databases with programming languages such as Python and JSON.
+
+**Pros** : Users mention that the tool has a large open source community that has built several plugins for the tool, improving its functionality.
+
+**Cons** : Users mention that it takes a long time to install and configure the solution.
+
+### Presentation softwares
+
+A Presentation Tool is a software package used to display data and information in the form of a slide show. The information is broken into small pieces called slides. It has three major parts: editor that allows formatting and editing of text, method used for inserting and manipulating graphic images, screenshots etc, slide-show system to display the content in form of a slide show.
+
+The play a vital role in all field as,
+
+- Digital presentations are excellent mediums for teaching activities, easy to create, use, copy store and transport.
+- The use of presentation software encourages use of all four language skills: Reading, Writing, Speaking and Listening.
+- Presentation software makes the ability to communicate messages to a group of people much simpler than any other delivery methods.
+- Seeing the presentation while hearing about it helps both visual and auditory viewers to absorb the information.
+
+The most common presentation tools are,
+
+- LibreOffice Impress
+- [WPS Presentation](https://24slides.com/presentbetter/top-10-free-presentation-tools-corporate-presenters/#3_WPS_Presentation)
+- [FreeOffice 2018&#39;s Presentation](https://24slides.com/presentbetter/top-10-free-presentation-tools-corporate-presenters/#4_FreeOffice_2018s_Presentation)
+- [Focusky](https://24slides.com/presentbetter/top-10-free-presentation-tools-corporate-presenters/#5_Focusky)
+- [SlideDog Free](https://24slides.com/presentbetter/top-10-free-presentation-tools-corporate-presenters/#6_SlideDog_Free)
```

Comment:
It looks like you found one presentation and added here. Try to add official links.